src/main/java/com/vortexa/refinery/exceptions/ManagedException.kt

```diff
@@ -31,8 +31,20 @@ abstract class ManagedException(override val message: String, val level: Level)
     }
 }
 
-class CellParserException(message: String) : ManagedException(message, Level.WARNING)
-
+class CellParserException : ManagedException {
+    constructor(message: String) : super(message, Level.WARNING)
+
+    constructor(
+        parserName: String,
+        columnName: String?,
+        columnNumber: Int?,
+        cellValue: String?
```

Comment:
I'd rather make them the values of the class instead of just passing the to the constructor to generate the message